modules/swagger-codegen/src/main/resources/JavaPlayFramework/LICENSE.mustache

```diff
@@ -0,0 +1,8 @@
+This software is licensed under the Apache 2 license, quoted below.
```

Comment:
no problem. I was just mimicking the output of a brand new Play Framework project.